README.md

```diff
@@ -273,6 +270,21 @@ In the CORS section of the file you can find the next options:
 - **allow_credentials** (boolean)
   
   Represent the **Access-Control-Allow-Credentials header** you must indicate if true or false
+  
+## Imposter configuration file 
+Imposter file consists of JSON array of imposters. Each imposter can have two sections: `request` that describes request and `response`.
+
+### Request section
+
+### Response section
+Response section describes response that imposter server will send on matching request. Response can have next fields:
+
```

Comment:
I don't understand this section, I think is not relevant at all.

Reply:
I've thought documentation need full description of imposter files anyway.

Reply:
Yes, you're right we're working on it, and we want to create an entire page to explain all application, and how to use each part, but for now, in we want to keep the README clear. So for now, follow my advice to try to merge your branch